src/leap/bitmask/gui/mainwindow.py

```diff
@@ -1089,10 +1089,9 @@ def _help(self):
 
         # FIXME on i3, this doens't allow to mouse-select.
         # Switch to a dialog in which we can set the QLabel
-        imap_token = (self._service_tokens.get('imap', None) or
-                      "??? (log in to unlock)")
-        smtp_token = (self._service_tokens.get('smtp', None) or
-                      "??? (log in to unlock)")
+        imap_token = smtp_token = (
+            self._service_tokens.get('mail_auth', None) or
+            "??? (log in to unlock)")
         imap_password = self.tr("IMAP Password:") + " %s" % (imap_token,)
         smtp_password = self.tr("SMTP Password:") + " %s" % (smtp_token,)
```

Comment:
`smtp_token` variable does not exist anymore. I think we should rename imap_token and use it in both places.